.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,34 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+#     - name: Lint with flake8
```

Comment:
To this it would also be nice to add https://github.com/timothycrosley/isort and https://github.com/psf/black.

However, `black` is Python 3 only, so need to be careful with that.

Example isort: `isort --recursive --check-only --diff newsapi tests`